prime-dotnet-webapi/Helpers/RequestLoggingMiddleware.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+
+namespace Prime.Helpers
+{
+    /// <summary>
+    /// Based on https://blog.elmah.io/asp-net-core-request-logging-middleware/
+    /// </summary>
+    public class RequestLoggingMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private readonly ILogger _logger;
+
+        /// <summary>
+        /// Called once in <c>Startup</c>
+        /// </summary>
+        /// <param name="next"></param>
+        /// <param name="loggerFactory"></param>
+        public RequestLoggingMiddleware(RequestDelegate next, ILoggerFactory loggerFactory)
+        {
+            _next = next;
+            _logger = loggerFactory.CreateLogger<RequestLoggingMiddleware>();
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            _logger.LogInformation(
+                "Incoming request:  {method} {url}",
+                context.Request?.Method,
+                context.Request?.Path.Value);
+
+            await _next(context);
```

Comment:
So the original AC writes: `All requests to the web-api are logged,` **`not just the responses`** `(if any)`
Does the response get logged anywhere in PRIME, if not should it be logged here as well?

Reply:
So currently in the `develop` branch, the responses are already logged by some mechanism I'm not familiar with.  In OpenShift:
 
![image](https://user-images.githubusercontent.com/7586665/153691768-9bf07a21-3572-41b0-82e4-1aa83f24e201.png)